Casks/openvox-agent-8.rb

```diff
@@ -0,0 +1,32 @@
+cask 'puppet-agent-8' do
```

Comment:
Should this line be openvox-agent-8?

Reply:
good catch! missed cask names, will fix now